### PR TITLE
Fix for #742

### PR DESCRIPTION
**Pull request by A-Aravindhan**

### CODE REVIEW

File: modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java [hunk 1]
```diff
@@ -362,6 +378,18 @@ public Server getServer(ObjectNode obj, String location, ParseResult result) {
 
         String value = getString("url", obj, true, location, result);
         if(StringUtils.isNotBlank(value)) {
+        	File file = new File(value);
+			if(!isValidURL(value) && !file.isAbsolute() && path != null){
```
Comment (jmini) [inline modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java]: The check `&& !file.isAbsolute()` is not necessary if you consider a server definition like this:

```json
      {
        "description": "Server relative to root path",
        "url": "/api/v2"
      }
```

If you remove the check, then you also can remove `File file = new File(value);`

**Comment by gracekarina** [comment]
Hi, @A-Aravindhan  @jmini to be able to merge this PR, can you please integrate your changes into one PR? Thanks!

**Comment by jmini** [comment]
I am not maintainer of this project, I can not update it: I do not have write permissions on `A-Aravindhan:v2.0.1` branch.

---

@gracekarina: 

Performing a quick [API call on this PR](https://api.github.com/repos/swagger-api/swagger-parser/pulls/754), I see that you have the permission: `maintainer_can_modify: true`

You need to add both forks as remote (assuming you work with ssh on GitHub). This should do it:

```
$ git remote add jmini git@github.com:jmini/swagger-parser.git
$ git remote add A-Aravindhan git@github.com:A-Aravindhan/swagger-parser.git
$ git fetch jmini pr754
$ git push A-Aravindhan jmini/pr754:v2.0.1
```

Let me know if it does not work.
Thank you in advance.

**Comment by jmini** [comment]
@gracekarina : I have opened a replacement PR that contains all the changes discussed here => #773

**Comment by gracekarina** [comment]
Hi, I just merged the #773 PR. Thanks for your contributions.

**Pull request merged**